sunpy/net/vso/tests/test_vso.py

```diff
@@ -350,3 +350,11 @@ def test_row_to_table(mocker, mock_build_client, client, mock_table_response):
     as_table = mocker.patch("sunpy.net.base_client.QueryResponseRow.as_table", return_value=[])
     client.fetch(mock_table_response[0])
     assert as_table.called
+
+
+@pytest.mark.remote_data
+def test_iris_filename(client):
+    search_results = client.search(a.Time("2018-01-02 15:31:55", "2018-01-02 15:31:55"), a.Instrument.iris)
+    # Downloads one file ~22MB in size.
```

Comment:
I probs should avoid this.

Reply:
Is there a way of inspecting the search results to check that the file to be downloaded ends in `.fits`?

Reply:
Good question, I do not know.